nodebuilder/settings.go

```diff
@@ -125,8 +175,7 @@ func initializeMetrics(
 			semconv.SchemaURL,
 			semconv.ServiceNamespaceKey.String(fmt.Sprintf("Celestia-%s", nodeType.String())),
 			semconv.ServiceNameKey.String(fmt.Sprintf("semver-%s", buildInfo.SemanticVersion)),
-			semconv.ServiceInstanceIDKey.String(peerID.String()),
-		)))
+			semconv.ServiceInstanceIDKey.String(fmt.Sprintf("%s/%s", network.String(), peerID.String())))))
```

Comment:
Lets keep only those attributes:
- network/node_type
- peer.ID

Reply:
what about "Celestia" prefix?

Reply:
@jrmanes , are u ok with @walldiss suggestion?

Reply:
I don't think we need celestia prefix either, just network name is enough and peer ID

Reply:
I think we should combine peer.ID and network, but keep nodeType separately

Reply:
I'd say network/peer.ID makes more sense to me, both are info related to the environment, and the node type can be separated, for me, it's more like if we want to filter by node types rather than useful info of the chain, but no problem if we want to do it in the other way

Reply:
Agree. `network/peer.ID` or `network:peer.ID` looks nicer imo

Reply:
`network:peer.ID` reminds me of the `CELESTIA_CUSTOM` env var format which is 👌 if we want to keep the same format

Reply:
left "/" , but I'm open to any suggestions

Reply:
The point of network/node type is to be able to filter data by specific combination. So I can see only robusta-64 bridge nodes lets say. 
I'm not sure what is the point of combining network and peer.ID. But would love to hear your filtering usecases that will use it.